Core/include/Acts/TrackFitting/KalmanFitter.hpp

```diff
@@ -46,7 +46,7 @@
 
 namespace Acts {
 
-/// Extension struct which holeds delegates to customize the KF behavior
+/// Extension struct which holds delegates to customise the KF behavior
```

Comment:
Currently, we have a happy mix, but the current [CERN English language style guide](https://translation-council-support-group.web.cern.ch/sites/default/files/styles/CERN%20TM%20English%20language%20style%20guide.pdf) reads "As a general rule, preference is given to British rather than US spelling".

Reply:
Don't think we should make changes just for this, and I don't think the language in our code even needs to be consistent.